### PR TITLE
feat(grid): add focus state styles for the column list item in the column menu

### PR DESCRIPTION
**Pull request by JoomFX**
fixes https://github.com/telerik/kendo-themes/issues/4655

### CODE REVIEW

File: tests/_output/fluent/grid/grid-column-menu.png
Comment (Juveniel) [inline tests/_output/fluent/grid/grid-column-menu.png:1]: @JoomFX the focus in the "Columns grouped expanded" scenario in Fluent does not look as expected. Could you please take a look and see if we can fix this ?

Comment (JoomFX) [reply tests/_output/fluent/grid/grid-column-menu.png:1]: @Juveniel I don't know if we have designs for this exact scenario, but I would expect the focus style to be the same as the one for the List item:

focus

In my opinion the focus style is expected.
Apart from the above, the focus styles for the Column List Item [were already present](https://github.com/telerik/kendo-themes/blob/develop/packages/fluent/scss/grid/_theme.scss#L490) in the Fluent theme.
Let me know if I'm missing something.

Comment (Juveniel) [reply tests/_output/fluent/grid/grid-column-menu.png:1]: image

Comment (Juveniel) [reply tests/_output/fluent/grid/grid-column-menu.png:1]: this is due to the scrollbar that is unfortunately invisible in the visual tests, thus not an actual issue

**Review by Juveniel — APPROVED** [review]
change commit to `fix`, otherwise lgtm

**Pull request merged**